src/common/hostProcess.ts

```diff
@@ -19,20 +21,19 @@ export enum HostProcessType {
     All = 3,      // bits: 11
 }
 
-import os from "os";
```

Comment:
I believe this is what caused some of your initial test issues... the os module and call to release was running inline within the module therefore making it really difficult to mock since it was always referenced an initialized before any tests were running.  Moved this call to inside the method and exported the method vs. the type.